index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="style.css" rel="stylesheet">
+        <title>Portfolio: Setup and Mobile Version Skeleton</title>
+    </head>
+    <body>
+        <header class="toolbar">
+            <a href="#" class="logo">Welcome</a>
+            <nav>
+                <button type="button" class="btn btn-tertiary"><img src="Images/Normal-Button/Tertiary/Icons/Union.svg" alt="union icon"></button>
+            </nav>
+        </header>
+        <section class="headline">
+            <h1>Hey there. I’m Sahar
+                <span class="h1-span">I’m a software developer</span>
+            </h1>
```

Comment:
- [REQUIRED] You did well implementing the changes requested by the previous code reviewer 👍 . However, you still need to separate your `title `and `subtitle `with `h1 `and `h3 `accordingly

Reply:
Regarding this issue,  I don't agree that should be separated into H1 and H3 since the both lines are part of the same title.
In addition, I believe that I followed all the guidelines and the PR shouldn't be rejected for it (Please note that the previous reviewer didn't separate it into h1 and h3 but wrote it as I did)

Reply:
OK thank you, I fixed it.